_ml-commons-plugin/cluster-settings.md

```diff
@@ -246,3 +276,23 @@ plugins.ml_commons.model_auto_redeploy_success_ratio: 0.8
 
 - Default value: 0.8
 - Value range: [0, 1]
+
+## Run Python-based models
+
+When set to `true`, this setting enables the ability to run Python-based models, such as [Metrics correlation]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms/#metrics-correlation).
```

Comment:
Is this accurate ? @dhrubo-os 

I think this setting only controls whether user can run the model built by OpenSearch team.

Reply:
Good point @ylwu-amzn. May be we can update it to: `this setting enables the ability to run Python-based models built by Opensearch team`